api/dingus.js

```diff
@@ -0,0 +1,17 @@
+const marked = require('../');
+const version = require('../package.json').version;
+const name = 'Marked.js';
```

Comment:
How do we want to brand ourselves here? We are inconsistent so far.

- The [README.md](https://github.com/markedjs/marked/blob/master/README.md) and [LICENSE.md](https://github.com/markedjs/marked/blob/master/LICENSE.md) use `Marked`
- The [documentation](https://github.com/styfle/marked/blob/master/docs/index.html#L5) uses `Marked.js`
- The [AUTHORS.md](https://github.com/markedjs/marked/blob/master/docs/AUTHORS.md#users) and our bot use `MarkedJS`

Let's pick one and use it everywhere.

cc @UziTech @joshbruce @davisjam

Reply:
I would say that the project would be: MarkedJS

The product itself would be: Marked

The distinction seems to be the way users reference Marked as a library - and make finding the GitHub Org easier to find.

Never: Marked.js

This combination also covers the repo itself: markedjs/marked

Reply:
I would just use marked seems like the js is overkill since there isn't a non js version of marked

Reply:
Thanks, I updated to `Marked` 👍

Reply:
@UziTech @joshbruce What do you think about moving our domain to Vercel? This would allow us to do https://marked.js.org/api/dingus?text=Hello with the canonical domain name.

Reply:
@UziTech & @styfle - Can we change our org name?? marked/marked 

(A naming convention I'm using with my on project is prefix the repo with the langaguage or framework. `php-`, `js-`, `ruby-`, `laravel-`, `angular-`, and so on. To UziTech's point - we don't have other language support though I do think some of the forks are in something like TypeScript; so, `ts-`...)

While I appreciate separating the org and product concepts, I do appreciate full consistency.

> https://marked.js.org/api/dingus?text=Hello

(Putting in quote to emphasize branding inconsistency. Users need to remember when to use JS and when not - understanding this is because we are a sub to js.org.)

Not sure I'm following the possible impact on the change. Appreciate the no downtime and, given I don't help maintain the site, trust your judgment on that score.

Reply:
> Not sure I'm following the possible impact on the change. 

Currently https://marked.js.org is hosted on GitHub Pages which is static only, we can't host an API. The purpose of my PR is to host an API. I suggested Vercel because our PR Preview deployments already go to Vercel (see the bot preview link in this PR).

Reply:
@styfle I don't see any down sides to moving to Vercel. It might make the demo site simpler if we could do some things on the server side (like build PRs to be able to use them in a demo).

@joshbruce I wouldn't mind changing the name to marked/marked, however I'm not sure it is necessary. 50/50 to me

Reply:
@joshbruce We can't rename the github org to `marked` because its already taken https://github.com/marked

And I think the github org discussion requires a separate thread anyway.

Please view my PR here since there is nothing else we need to change at this time.